src/backend/distributed/shardsplit/shardsplit_logical_replication.c

```diff
@@ -0,0 +1,734 @@
+/*-------------------------------------------------------------------------
+ *
+ * shardsplit_logical_replication.c
+ *
+ * Function definitions for logically replicating split children.
```

Comment:
nitpick: logically replication shard to split children.

Reply:
Same fix in header.

Reply:
Done. Thanks.